runners/console/index.ts

```diff
@@ -410,10 +411,60 @@ export class Console extends Runner {
         if(!fs.existsSync(workspacesDir))
             fs.mkdirSync(workspacesDir);
         this.setVariable(this.workspaceDirectory, workspacesDir);
+     
+      return result;
+    }
+
+    runExecuteCommand(runCommand: RunCommand): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0; 
+        
+        let exeCommand = (runCommand.command.parameters.length > 1 && runCommand.command.parameters[1].args)
+        ? runCommand.command.parameters[0]+ " " +runCommand.command.parameters[1].args.join(" ")
+        : runCommand.command.parameters[0];
+
+        let dirPath = (runCommand.command.parameters.length > 1 && runCommand.command.parameters[1].dir)
+        ? path.join(this.getVariable(this.workspaceDirectory), runCommand.command.parameters[1].dir)
+        : this.getVariable(this.workspaceDirectory)
+
+        if(runCommand.command.parameters.length > 1 && runCommand.command.parameters[1].asynchronous){
+            let port = runCommand.command.parameters[2].port 
+            ? runCommand.command.parameters[2].port
```

Comment:
Why specify a default port and not just throw an error message? We should do a refactoring, if this approach is the better choice.

Reply:
I was thinking about doing it but in the last Review, nobody replied to that https://github.com/devonfw-tutorials/tutorial-compiler/pull/154#discussion_r609577288 so I just used @MarcelDiessner Idea of taking a default port. There still be an Error, when the assert Methode checks for a given port and will throw an Error. The default port is just for killing the process.

Reply:
The port will be different in most tutorials. It depends on the application or server you start with the command. You can't just say this is now running on port xyz.
I think the user has to specify this in their tutorials and as Denise said, throw an error if no port is specified. Because there is no way to find out what port the application is running on.

Reply:
In my opinion throwing an error would be better. I don't think, that the background process will be found, based on the default port and thus won't be killed in the end, but I could be wrong.